Glossary of Terms.md

```diff
@@ -6,9 +6,9 @@ A brief list of terms as used in the [API Blueprint](http://apiblueprint.org) co
 
 <a name="def-action"></a>
 ### Action
-An **HTTP transaction** (a request-response transaction).
+One or more [**HTTP Transactions**](#def-transaction) within a [Resource](#def-resource) with a common [HTTP Request Method](#def-method).
```

Comment:
I understood one HTTP Transaction as a single pair of one Request and one Response. Action can contain multiple Requests and Responses in various combinations representing in the end multiple HTTP Transactions.

Maybe my understanding is wrong?